### PR TITLE
bpo-41851: tkinter font add neg and equal methods

### PR DESCRIPTION
**Pull request by E-Paine**
https://bugs.python.org/issue41851

### CODE REVIEW

File: Lib/tkinter/font.py [hunk 1]
```diff
@@ -118,6 +118,11 @@ def __del__(self):
         except Exception:
             pass
 
+    def __neg__(self):
+        cp = self.copy()
+        cp['size'] = -(cp['size'])
```
Comment (gareth-rees) [inline Lib/tkinter/font.py]: Parentheses are not necessary here: indexing/slicing is higher precedence than unary negation: see the [operator precedence table](https://docs.python.org/3/reference/expressions.html#operator-precedence).

File: Doc/library/tkinter.font.rst [hunk 2]
```diff
@@ -61,6 +61,13 @@ The different font weights and slants are:
 
       Return new instance of the current font.
 
+   .. method:: equal(other)
+
+      Compare the font to another and check if they represent the same font (in
+      contrast to ``__eq__``, which checks if they are the same Tk object).
```
Comment (gareth-rees) [inline Doc/library/tkinter.font.rst]: Write this as:
```
:meth:`__eq__`
```
to get consistent formatting of method names.

File: Doc/library/tkinter.font.rst [hunk 3]
```diff
@@ -83,6 +90,13 @@ The different font weights and slants are:
 
       *fixed* - 1 if font is fixed-width else 0
 
+   .. method:: __neg__()
+
+      Returns a new font object with the size negated. In practice, this means
```
Comment (gareth-rees) [inline Doc/library/tkinter.font.rst]: Use the imperative mood:

> Return a new font object ...

File: Lib/tkinter/test/test_tkinter/test_font.py [hunk 4]
```diff
@@ -72,6 +72,24 @@ def test_eq(self):
         self.assertNotEqual(font1, 0)
         self.assertEqual(font1, ALWAYS_EQ)
 
+    def test_neg(self):
+        font1 = font.Font(root=self.root, size=32)
+        font2 = -font1
+        self.assertEqual(font1['size'], -(font2['size']))
```
Comment (gareth-rees) [inline Lib/tkinter/test/test_tkinter/test_font.py]: This is the same as the next line, as the parentheses are redundant.

**Comment by E-Paine** [comment]
@gareth-rees thanks for the review! I missed the point about operator precedence: thank you for pointing this out.

**Comment by E-Paine** [comment]
As Serhiy does not see this method [worth pursuing](https://bugs.python.org/issue41851#msg377602), I am closing this PR to instead potentially work on improving the docs for the existing `actual` method.

**Pull request closed without merging**